examples/standalone/main.tf

```diff
@@ -0,0 +1,203 @@
+  dlp_location              = local.location
+  template_id_prefix        = "de_identification"
+  template_file             = "${path.module}/templates/deidentification.tmpl"
+  dataflow_service_account  = module.secured_data_warehouse.dataflow_controller_service_account_email
+}
+
+module "re_identification_template" {
+  source = "../..//modules/de-identification-template"
+
+  project_id                = module.base_projects.data_governance_project_id
+  terraform_service_account = var.terraform_service_account
+  crypto_key                = module.tek_wrapping_key.keys[local.kek_key_name]
+  wrapped_key               = local.wrapped_key_secret_data
+  dlp_location              = local.location
+  template_id_prefix        = "re_identification"
+  template_file             = "${path.module}/templates/reidentification.tmpl"
+  dataflow_service_account  = module.secured_data_warehouse.confidential_dataflow_controller_service_account_email
+}
+
+resource "google_artifact_registry_repository_iam_member" "docker_reader" {
+  provider = google-beta
+
+  project    = module.template_project.project_id
+  location   = local.location
+  repository = "flex-templates"
+  role       = "roles/artifactregistry.reader"
+  member     = "serviceAccount:${module.secured_data_warehouse.dataflow_controller_service_account_email}"
+}
+
+resource "google_artifact_registry_repository_iam_member" "confidential_docker_reader" {
+  provider = google-beta
+
+  project    = module.template_project.project_id
+  location   = local.location
+  repository = "flex-templates"
+  role       = "roles/artifactregistry.reader"
+  member     = "serviceAccount:${module.secured_data_warehouse.confidential_dataflow_controller_service_account_email}"
+}
+
+resource "google_artifact_registry_repository_iam_member" "python_reader" {
+  provider = google-beta
+
+  project    = module.template_project.project_id
+  location   = local.location
+  repository = "python-modules"
+  role       = "roles/artifactregistry.reader"
+  member     = "serviceAccount:${module.secured_data_warehouse.dataflow_controller_service_account_email}"
+}
+
+resource "google_artifact_registry_repository_iam_member" "confidential_python_reader" {
+  provider = google-beta
+
+  project    = module.template_project.project_id
+  location   = local.location
+  repository = "python-modules"
+  role       = "roles/artifactregistry.reader"
+  member     = "serviceAccount:${module.secured_data_warehouse.confidential_dataflow_controller_service_account_email}"
+}
+
+// The sample data we are using is a Public Bigquery Dataset Table
+// that contains a United States Internal Revenue Service form
+// that provides the public with financial information about a nonprofit organization
+// (https://console.cloud.google.com/bigquery?project=bigquery-public-data&d=irs_990&p=bigquery-public-data&page=table&ws=!1m9!1m3!3m2!1sbigquery-public-data!2sirs_990!1m4!4m3!1sbigquery-public-data!2sirs_990!3sirs_990_ein&t=irs_990_ein)
```

Comment:
maybe point to the README or use the same suggested link.  This link seems wrong